.github/workflows/nightly_release.yml

```diff
@@ -0,0 +1,67 @@
+name: SalesforceCommerceCloud/pwa-kit/nightly_release
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 12am (PST) - cron uses UTC times
+    - cron: "0 8 * * *"
+
+jobs:
+  create_nightly_release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get Timestamp
+        run: |-
+          echo "release_timestamp=$(date +'%m%d%Y%H%M%S')" >> "$GITHUB_ENV"
+
+      - name: Get monorepo version
+        run: |-
+          version=`cat package.json | jq -r ".version" | cut -d "-" -f 1`
+          echo "monorepo_version_base=$version" >> "$GITHUB_ENV"
+
+      - name: Get retail-react-app version
+        run: |-
+          version=`cat packages/template-retail-react-app/package.json | jq -r ".version" | cut -d "-" -f 1`
+          echo "retail_app_version_base=$version" >> "$GITHUB_ENV"
+
+      - name: Get commerce-sdk-react version
+        run: |-
+          version=`cat packages/commerce-sdk-react/package.json | jq -r ".version" | cut -d "-" -f 1`
+          echo "commerce_sdk_react_version_base=$version" >> "$GITHUB_ENV"
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: npm
+
+      - name: Install Monorepo Dependencies
+        run: |-
+          # Install node dependencies
+          node ./scripts/gtime.js monorepo_install npm ci
+
+      - name: Create Release Branch
+        run: git checkout -b release-${{ env.monorepo_version_base }}-nightly-${{ env.release_timestamp }}
+
+      - name: Bump version (monorepo)
```

Comment:
Does the Bump version script already accounts for the nightly releases to update the NPM `latest` and next `tag`?

What happens if a regular release last more than one day? Are the NPM tags going to be overwritten by the nightly build?

Should we use a dedicated NPM tag like `nightly-next` or `nightly-latest` to avoid colliding with the regular release process?

Reply:
I think it's ok to use the tag `next` and understand that this tag might need to manually changed if we release a broken nightly

Reply:
@adamraya I updated the push-to-npm.js script to account for nightly releases and manage npm tags accordingly